### DISCUSSION TITLE
Add `quiet` mode option to various commands

### DISCUSSION THREAD

**Discussion by Drew-Daniels** [ankitpokhrel/jira-cli#802] [Ideas]
Thanks for building this CLI! It's been really useful for building out some CLI automations to help with my daily workflow.

Many CLI's have a `--quiet` flag that can be used with various commands to suppress output logged to `stdout`, but it doesn't look like any of the `jira-cli` commands have this option.

I think adding a flag for this functionality would be useful for people developing things like shell scripts that use the `jira-cli` internally, and don't want output from the CLI to be output into the user's terminal.

An example is when I run the `jira list issues` command inside of a shell script. If no issues are returned, the `✗ No result found for given query in project "<PROJECT-IDENTIFIER>"` is always printed.

We can work around this be redirecting the output, but I think it'd be nice if there were an option in the CLI just to suppress output in a less hacky way.

**Comment by etosan** [comment]
It would be really helpful for me and my clients too, `--quiet` could be the primary option for main command, not for specific sub-command and it should propagate to any sub-commands in chain automatically.

Most important effect should be suppressing "Fetching..." "throbber" TUI animation - it is cute at first, but when scripting or doing `--plain` work it gets annoying pretty fast and it just pollutes logs and terminal.
